.github/workflows/test.yaml

```diff
@@ -98,6 +98,7 @@ jobs:
       run: |
         cd ${{ github.workspace }}/src/github.com/snapcore/snapd
         git fetch origin ${{ github.base_ref }}:${{ github.base_ref }}
+        git fetch origin ${{github.ref}}
```

Comment:
Do we need this? The CLA check seems to not have it and does something similar(?)

Reply:
I saw this branch work successfully in one run and then rebased to squash my commits and now the branch no longer works so I need to debug this branch more, but _something_ needs to be done because the exact same commands locally that are being run in GitHub Actions work perfectly locally :-(

Reply:
ah what we need is this I think:

https://github.com/jhenstridge/snapd/blob/e4720ca1f7842e700058ab409ecfd740b9672659/.github/workflows/cla-check.yaml#L18-L20

we do not have that on master

Reply:
added that fetch-depth and now it is working correctly